api/src/test/java/uk/gov/dhsc/htbhf/claimant/message/context/MessageContextLoaderTest.java

```diff
@@ -355,4 +355,28 @@ void shouldSuccessfullyLoadNewCardContext() {
         verify(claimRepository).findById(claimId);
     }
 
+    @Test
+    void shouldSuccessfullyLoadAdditionalPregnancyPaymentContext() {
+        //Given
+        Claim claim = aValidClaim();
+        UUID claimId = claim.getId();
+        PaymentCycle latestPaymentCycle = aPaymentCycleWithClaim(claim);
+        given(claimRepository.findById(any())).willReturn(Optional.of(claim));
+        given(paymentCycleRepository.findCurrentCycleForClaim(any())).willReturn(Optional.of(latestPaymentCycle));
+        var payload = AdditionalPregnancyPaymentMessagePayload.builder().claimId(claimId).build();
```

Comment:
Shouldn't be using `var` here.

Reply:
Does this warrant a Test Data Factory? We might need one when you build on this in the next PR(s).

Reply:
I don't seem the harm here when we can see that we're calling build on the same line. It's particular nice when you have very long class names.